Orange/projection/base.py

```diff
@@ -6,7 +6,21 @@
 from Orange.misc.wrapper_meta import WrapperMeta
 import Orange.preprocess
 
-__all__ = ["Projector", "Projection", "SklProjector"]
+__all__ = ["LinearCombinationSql", "Projector", "Projection", "SklProjector"]
+
+
+class LinearCombinationSql:
```

Comment:
I don't know what this actually does, but I suppose that this is a general linear combination, and it also supports SQL -- and not a linear combination that requires SQL, right? If so, the name is misleading. @lanzagar, could you help here?

Reply:
This is the sql equivalent of the compute_value. It only works for sql, as in-memory data is processed with a different object - Projector.

Reply:
Ah, OK, I see. Thanks.